app/javascript/mastodon/features/status/components/detailed_status.js

```diff
@@ -276,8 +288,9 @@ class DetailedStatus extends ImmutablePureComponent {
           {media}
 
           <div className='detailed-status__meta'>
-            <a className='detailed-status__datetime' href={`/@${status.getIn(['account', 'acct'])}\/${status.get('id')}`} target='_blank' rel='noopener noreferrer'>
+            <a className='detailed-status__datetime' href={`${status.get('url')}`} target='_blank' rel='noopener noreferrer' title='${intl.formatMessage(messages.canonical_link)}' aria-label='${intl.formatMessage(messages.canonical_link)}'>
```

Comment:
Hmmm I'm not too sure about adding a label/title here.

Reply:
My thinking is that you can hover over the timestamp, so you should be able to see a hint about what it leads to? The external icon is just part of the link at the end

Reply:
I really don't think `aria-label` is appropriate here, the thing's primary purpose is that it's a timestamp, it's not a visual-only thing either, I think `aria-label` would incorrectly obscure the timestamp.

Reply:
Would a `title` but no `aria-label` be appropriate, or should there be a different `aria-label`?